website/src/pages/showcase/index.js

```diff
@@ -19,6 +20,31 @@ const DESCRIPTION =
   'See the awesome websites people are building with Docusaurus';
 const EDIT_URL =
   'https://github.com/facebook/docusaurus/edit/master/website/src/data/users.js';
+const TAGS = [
+  'highlight',
+  'design',
+  'i18n',
+  'versioning',
+  'multi-instance',
+  'large',
+  'facebook',
+  'personal',
+  'rtl',
+];
+let selectedTags = [];
+let filteredUsers = users;
```

Comment:
That can't work this way, UI state in React must be stored as React state, otherwise React does not know when to re-render

ie using `const [selectedTags,setSelectedTags] = useState([])`